images/linux/scripts/installers/Install-AzureModules.ps1

```diff
@@ -17,6 +17,25 @@ foreach ($module in $modules)
         Write-Host " - $version [$modulePath]"
         Save-Module -Path $modulePath -Name $moduleName -RequiredVersion $version -Force -Verbose
     }
+
+    $assets = Invoke-RestMethod $module.url
+
+    # Get github release asset for each version
+    foreach ($toolVersion in $module.zipversions) {
+        $asset = $assets | Where-Object version -like $toolVersion `
+        | Select-Object -ExpandProperty files `
+        | Where-Object { ($_.platform -eq $module.platform) -and ($_.platform_version -eq $module.platform_version)} `
```

Comment:
I think we don't need this condition because versions-manifest.json doesn't contain any platforms and platform_version values.